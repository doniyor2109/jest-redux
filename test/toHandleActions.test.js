import { reducer, pureActions } from './stub';

describe('toHandleActions', () => {
  it('should match with previous results', () => {
    expect(reducer).toHandleActions(pureActions);
  });
});