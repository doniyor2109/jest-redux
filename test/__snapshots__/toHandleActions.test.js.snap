// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`toHandleActions should match with previous results: add should match with snapshot 1`] = `1`;

exports[`toHandleActions should match with previous results: remove should match with snapshot 1`] = `-1`;
